### PR TITLE
OCPBUGS-35488_4.14#updating installation doc

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Version:
4.14

Issue:
https://issues.redhat.com/browse/OCPBUGS-35488

Link to docs preview:

**AWS**
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations#installation-aws-tested-machine-types_installing-aws-customizations
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations#installation-aws-arm-tested-machine-types_installing-aws-customizations

**Azure**
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-tested-machine-types_installing-azure-customizations 
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations#installation-azure-arm-tested-machine-types_installing-azure-customizations

**GCP**
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations#installation-gcp-tested-machine-types_installing-gcp-customizations
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations#installation-gcp-tested-machine-types-arm_installing-gcp-customizations 

**IBM Cloud**
- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations#installation-ibm-cloud-tested-machine-types_installing-ibm-cloud-customizations

- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations#installation-ibm-cloud-tested-machine-types_installing-ibm-cloud-network-customizations 

- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc#installation-ibm-cloud-tested-machine-types_installing-ibm-cloud-vpc

- https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-private#installation-ibm-cloud-tested-machine-types_installing-ibm-cloud-private

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@jinyunma, PTAL

**Comment by MayXuQQ** [comment]
@brendan-daly-red-hat where is the IBMCloud review link?

**Comment by jinyunma** [comment]
@brendan-daly-red-hat seems that the link in following adocs (azure-tested-machines-types for x86 64-bit) has not be updated yet.
[installation-azure-tested-machine-types.adoc](https://github.com/openshift/openshift-docs/blob/enterprise-4.14/modules/installation-azure-tested-machine-types.adoc)

File: modules/installation-ibm-cloud-tested-machine-types.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc.adoc
+
+[id="installation-ibm-cloud-tested-machine-types_{context}"]
```
Comment (jinyunma) [inline modules/installation-ibm-cloud-tested-machine-types.adoc:9]: Checked that section `Tested instance types for IBM Cloud` is added starting from 4.15 doc, https://docs.openshift.com/container-platform/4.15/installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.html#installation-ibm-cloud-tested-machine-types_installing-ibm-cloud-customizations

Do we need this adoc on 4.14 release? cc: @MayXuQQ

Comment (MayXuQQ) [reply modules/installation-ibm-cloud-tested-machine-types.adoc:9]: I prefer to add this section to 4.14 release

**Comment by bscott-rh** [comment]
> @brendan-daly-red-hat seems that the link in following adocs (azure-tested-machines-types for x86 64-bit) has not be updated yet. [installation-azure-tested-machine-types.adoc](https://github.com/openshift/openshift-docs/blob/enterprise-4.14/modules/installation-azure-tested-machine-types.adoc)

I'm not sure what you mean, it has been updated in this PR: https://github.com/openshift/openshift-docs/pull/82043/files#diff-890a137bab6656aa0c6731924eee152b87beb40b16789bfe2bb4b0ebb4b8a9a0

**Comment by bscott-rh** [comment]
> @brendan-daly-red-hat where is the IBMCloud review link?

@MayXuQQ It looks like the IBM cloud tested instance types list only exists in 4.16+. What version would you like us to include in the 4.14 and 4.15 documentation? Or can you backport the instance types list to the previous versions for us to include?

**Comment by jinyunma** [comment]
> > @brendan-daly-red-hat seems that the link in following adocs (azure-tested-machines-types for x86 64-bit) has not be updated yet. [installation-azure-tested-machine-types.adoc](https://github.com/openshift/openshift-docs/blob/enterprise-4.14/modules/installation-azure-tested-machine-types.adoc)
> 
> I'm not sure what you mean, it has been updated in this PR: https://github.com/openshift/openshift-docs/pull/82043/files#diff-890a137bab6656aa0c6731924eee152b87beb40b16789bfe2bb4b0ebb4b8a9a0

@bscott-rh thanks for checking, the content is correct now for azure-tested-machine-types, @brendan-daly-red-hat updated it in commit https://github.com/openshift/openshift-docs/compare/9f902f1289cbb119918190aff1c708b20d6a2d54..3e6aa21763a8fe8c3c474e5bcb103f5bf8efb7ae

**Comment by MayXuQQ** [comment]
waiting for PR https://github.com/openshift/installer/pull/9032(4.15) to be merged, then I will backport it to 4.14

**Comment by brendan-daly-red-hat** [comment]
@jianli-wei, Can you review the changes for GCP?
@yunjiang29, Can you review the changes for AWS?

File: modules/installation-aws-arm-tested-machine-types.adoc [hunk 2]
```diff
@@ -16,11 +16,11 @@ The following Amazon Web Services (AWS) 64-bit ARM instance types have been test
...
-Use the machine types included in the following charts for your AWS ARM instances. If you use an instance type that is not listed in the chart, ensure that the instance size you use matches the minimum resource requirements that are listed in "Minimum resource requirements for cluster installation". 
+Use the machine types included in the following charts for your AWS ARM instances. If you use an instance type that is not listed in the chart, ensure that the instance size you use matches the minimum resource requirements that are listed in "Minimum resource requirements for cluster installation".
 ====
 
-.Machine types based on 64-bit ARM architecture 
+.Machine types based on 64-bit ARM architecture
 [%collapsible]
 ====
-include::https://raw.githubusercontent.com/openshift/installer/master/docs/user/aws/tested_instance_types_aarch64.md[]
+include::https://raw.githubusercontent.com/openshift/installer/release-4.14/docs/user/aws/tested_instance_types_aarch64.md[]
```
Comment (jianli-wei) [inline modules/installation-aws-arm-tested-machine-types.adoc:25]: FYI https://github.com/openshift/installer/blob/release-4.14/docs/user/aws/tested_instance_types_aarch64.md also list `r8g.*`, while the preview [link](https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations#installation-aws-arm-tested-machine-types_installing-aws-customizations) only lists `c6g.*` and `m6g.*`.

Comment (jianli-wei) [reply modules/installation-aws-arm-tested-machine-types.adoc:25]: LGTM for other parts, thanks!

**Comment by brendan-daly-red-hat** [comment]
@MayXuQQ, PTAL at the IBM Cloud preview links.

File: modules/installation-ibm-cloud-tested-machine-types.adoc (hunk 1 above)
Comment (bscott-rh) [inline modules/installation-ibm-cloud-tested-machine-types.adoc:9]:
```suggestion
:_mod-docs-content-type: REFERENCE
[id="installation-ibm-cloud-tested-machine-types_{context}"]
```
please add the "reference" content type to each of the tested machine types modules right before the `id` line.

**Review by bscott-rh — APPROVED** [review]
One small change, otherwise LGTM.

**Pull request merged**